client/src/components/PixelButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './PixelButton.css';
+
+function PixelButton({ title, type, size }) {
+  return (
+    <div className='test'>
+      <button
+        className={
+          'pixel' +
+          (type === 'danger' ? ' danger' : ' primary') +
```

Comment:
ơ con này a tưởng cứ + thẳng type vào chứ cần gì so sánh nhỉ :-?

Reply:
con button này em tự code anh ơi :v

Reply:
thì kiểu như className={ 'pixel ' + type + ' ' + size}

Reply:
ko cho biến vào luôn đc ở đấy ạ :v em tìm thì thấy ng ta làm thế này thôi ạ

Reply:
Line 7:34:  Unexpected string concatenation of literals  no-useless-concat 
nó warning thê này ạ

Reply:
toang nhỉ, gặp cái > 2 biến phát là hơi dài dòng